simapp/test_helpers.go

```diff
@@ -505,3 +508,16 @@ type EmptyAppOptions struct{}
 func (ao EmptyAppOptions) Get(o string) interface{} {
 	return nil
 }
+
+// ModuleAccountAddrs provides a list of blocked module accounts from configuration in app.yaml
+//
+// Ported from SimApp
+func ModuleAccountAddrs() map[string]bool {
+	cfg := appconfig.LoadYAML(AppConfigYaml)
+	var bk bankkeeper.Keeper
+	err := depinject.Inject(cfg, &bk)
```

Comment:
```suggestion
	var bk bankkeeper.Keeper
	err := depinject.Inject(AppConfig, &bk)
```

No need to re-read the yaml. I think this is the pattern we should encourage